src/components/StyleGuide/ColorCard.tsx

```diff
@@ -871,23 +959,21 @@ export const makeColorCard = function (data) {
       dataBrandPrimaryColor={dataBrandPrimaryColor}
       dataBrandSecondaryColor={dataBrandSecondaryColor}
       notes={notes}
+      showColorDataTable={showColorDataTable}
       textColor={textColor}
       key={name}
     />
   );
   return card;
 };
-export const getColorCards = (category) => {
+export const getColorCards = (category: string) => {
   const cards = [];
-  const catArr = cssVars[category];
-  if (category === "researchLibraries") {
-    console.log(catArr);
-  }
-  for (let i = 0; i < catArr.length; i++) {
-    const colorData = catArr[i];
+  const catArr = colorCardsData[category];
+  catArr.map((color: any) => {
```

Comment:
`.map` returns an array so a better option is `catArr.forEach`.

But, because it returns an array, you can set `cards` to the returned value

```
const cards = catAr?.length < 0 ? [] : catArr.map((color: any) => makeColorCard(color));
```

instead of pushing the value `makeColorCard(colorData)` onto a new variable that is used in the `cards` array, you can return it directly.